docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,119 @@
+# Release 365 (XX Dec 2021)
+
+## General
+
+* Add support for [`TRUNCATE TABLE`](/sql/truncate). ({issue}`8921`)
+* Add support for time travel queries. ({issue}`8773`)
+* Add {func}`contains` to check whether a CIDR contains an IP address. ({issue}`9654`)
+* Report connector metrics in `EXPLAIN ANALYZE VERBOSE`. ({issue}`9858`)
+* Allow executing ``INSERT`` or ``DELETE`` statements on tables restricted with a row filter. ({issue}`8856`)
+* Improve performance of inequality joins. ({issue}`9307`)
+* Improve join performance for joins with small build side. ({issue}`9851`)
+* Make query final statistics more accurate. ({issue}`9888`, {issue}`9913`)
+* Improve query planning performance for queries containing large `IN` predicates. ({issue}`9874`)
+* Fix incorrect results when casting bigint values to bounded varchar type. ({issue}`552`)
+* Fix ``PREPARE`` statements for ``DROP/INSERT`` when table or schema name contains special characters. ({issue}`9823`)
+* Fix minor memory leak when queries are abandoned during the initial query submission phase. ({issue}`9962`)
+
+## Security
+
+* Allow configuring HTTP proxy for OAuth2 authentication. ({issue}`9920`, {issue}`10069`)
+* Add group-based and owner-based query access rules to file based system access control. ({issue}`9811`)
+* Use internal names for discovery client when automatic TLS is enabled for internal communications. This allows `discovery.uri` 
+  to be configured using a normal DNS name like `https://coordinator.trino`, and still use automatic TLS certificates. ({issue}`9821`)
+* Fix internal communication automatic TLS on Java 17. ({issue}`9821`)
+
+## BigQuery connector
+
+* Support reading `bignumeric` type whose precision is less than or equal to 38. ({issue}`9882`)
```

Comment:
```suggestion
* Support reading ``BIGNUMERIC`` type whose precision is less than or equal to 38. ({issue}`9882`)
```

Reply:
We've been using lower case to refer to data types